### PR TITLE
Substitute links to improved docs.

### PR DESCRIPTION
**Pull request by jkisk**
Replace links to API docs with links to resource docs.  Initially this pull request includes blog posts and guides, I will continue to update other content in a new PR.

Issue #2887

### CODE REVIEW

File: content/blog/2019-year-at-a-glance/index.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ It's impossible to highlight everything, but here are some of our favorite miles
...
.../intro/cloud-providers/aiven" >}}), [CloudAMQP]({{< relref "/docs/intro/cloud-providers/cloudamqp" >}}), [Cloudflare]({{< relref "/docs/intro/cloud-providers/cloudflare" >}}), [Consul]({{< relref "/docs/intro/cloud-providers/consul" >}}), [Datadog]({{< relref "/docs/intro/cloud-providers/datadog" >}}), [DNSimple]({{< relref "/docs/intro/cloud-providers/dnsimple" >}}), [Docker]({{< relref "/docs/intro/cloud-providers/docker" >}}), [Fastly]({{< relref "/docs/intro/cloud-providers/fastly" >}}), [F5 BigIP]({{< relref "docs/intro/cloud-providers/f5bigip" >}}), [GitLab]({{< relref "docs/intro/cloud-providers/gitlab" >}}), [Kafka]({{< relref "docs/intro/cloud-providers/kafka" >}}), [MySQL]({{< relref "docs/intro/cloud-providers/mysql" >}}), [New Relic]({{< relref "docs/intro/cloud-providers/newrelic" >}}), [Okta]({{< relref "docs/intro/cloud-providers/okta" >}}), [PostgreSQL]({{< relref "docs/intro/cloud-providers/postgresql" >}}), [RabbitMQ]({{< relref "docs/intro/cloud-providers/rabbitmq" >}}), [SignalFX]({{< relref "docs/intro/cloud-providers/signalfx" >}}), [Spotinst]({{< relref "docs/intro/cloud-providers/spotinst" >}}), and [Vault]({{< relref "docs/intro/cloud-providers/vault" >}}).
```
Comment (praneetloke) [inline content/blog/2019-year-at-a-glance/index.md:52]: I think all `relref` values must start with a `/`. cc @cnunciato 

For example, this in line `[Kafka]({{< relref "docs/intro/cloud-providers/kafka" >}})` should be `[Kafka]({{< relref "/docs/intro/cloud-providers/kafka" >}})`

File: content/blog/announcing-pulumi-0.15-kubernetes-cicd-openstack-and-more/index.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ const kibana = new helm.v2.Chart("kibana", {
 
 Check out the [Kubernetes
 overview]({{< relref "/docs/get-started/kubernetes" >}}) docs, the [API
-documentation]({{< relref "/docs/reference/pkg/nodejs/pulumi/kubernetes" >}})
+documentation]({{< relref "/docs/reference/pkg/nodejs/pulumi/kubernetes" TODO NEW K8s doc link>}})
```
Comment (praneetloke) [inline content/blog/announcing-pulumi-0.15-kubernetes-cicd-openstack-and-more/index.md:90]: Kubernetes is now available here: `/docs/reference/pkg/kubernetes`. It was merged last night.

File: content/blog/announcing-pulumi-0.15-kubernetes-cicd-openstack-and-more/index.md [hunk 3]
```diff
@@ -194,7 +194,7 @@ const instance = new os.compute.Instance("test", {
 exports.instanceIP = instance.accessIpV4;
 ```
 
-Check out the [API documentation]({{< relref "/docs/reference/pkg/nodejs/pulumi/openstack" >}})
+Check out the [Resource documentation]({{< relref "docs/intro/cloud-providers/openstack/" >}})
```
Comment (praneetloke) [inline content/blog/announcing-pulumi-0.15-kubernetes-cicd-openstack-and-more/index.md:197]: In this context, I think we should link to `/docs/reference/pkg/openstack/` rather than the intro page?

But again whichever one we choose, that is, either linking to the resource docs page under `/docs/reference/pkg...` or `docs/intro/cloud-providers/...` we should be consistent throughout this page.

File: content/blog/aws-cloudwatch-made-easy-with-pulumi-infrastructure-as-code/index.md [hunk 4]
```diff
@@ -94,7 +94,7 @@ taking. With Pulumi's
 [Crosswalk]({{< relref "/docs/guides/crosswalk/aws" >}}) APIs it's simple
 to get at this information. Inside all the main "Crosswalk for AWS"
 modules are exposed
-[Metrics]({{< relref "/docs/reference/pkg/nodejs/pulumi/awsx/cloudwatch#Metric" >}})
+[Metrics]({{< relref "/docs/guides/crosswalk/aws/cloudwatch/#subscribing-to-cloudwatch-metrics" >}})
```
Comment (praneetloke) [inline content/blog/aws-cloudwatch-made-easy-with-pulumi-infrastructure-as-code/index.md:97]: We should definitely keep the link to the `awsx` and not change it to `aws`. The reason is, `awsx` is an extension package on top of `aws`. However, we will not have the new schema-based resource docs for `awsx`. So I don't think we need to change this link from what it is currently.

The same goes for the following packages as well:

* `policy`
* `kubernetesx` (note the `x` here.)
* `eks`
* `pulumi/pulumi`

File: content/blog/aws-cloudwatch-made-easy-with-pulumi-infrastructure-as-code/index.md [hunk 5]
```diff
@@ -137,7 +137,7 @@ of the health of this app:
         ],
     });
 
-The [Crosswalk for AWS API for dashboards](https://github.com/pulumi/pulumi-awsx/tree/master/nodejs/awsx/cloudwatch#dashboards)
+The [Crosswalk for AWS API for dashboards]({{< relref "/docs/guides/crosswalk/aws/cloudwatch/#defining-cloudwatch-dashboards-in-code" >}})
```
Comment (praneetloke) [inline content/blog/aws-cloudwatch-made-easy-with-pulumi-infrastructure-as-code/index.md:140]: I am not sure if we want to change the link here. The link text seems to specifically talk about the API for dashboards. If we do want to link to the guide instead, that's fine. Just making sure that that is what we should be doing.

File: content/blog/2019-year-at-a-glance/index.md (hunk 1 above)
Comment (justinvp) [reply content/blog/2019-year-at-a-glance/index.md:52]: Yes, please start with `/` to avoid ambiguity.

File: content/blog/aws-cloudwatch-made-easy-with-pulumi-infrastructure-as-code/index.md (hunk 4 above)
Comment (jkisk) [reply content/blog/aws-cloudwatch-made-easy-with-pulumi-infrastructure-as-code/index.md:97]: Oh great, this addresses some other questions I had as well :).

File: content/blog/adopting-existing-cloud-resources-into-pulumi/index.md [hunk 6]
```diff
@@ -24,15 +24,15 @@ We'll review referencing existing resources, and then dive deeper into how you c
 
 For referencing existing resources, Pulumi offers several tools.
 
-* The `.get` methods available on every resource let you [get all the details for a resource]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#Vpc-get" >}}) from the cloud provider based just on its `id`.
+* The `.get` methods available on every resource let you [get all the details for a resource]({{< relref "/docs/reference/pkg/aws/ec2/vpc/#look-up-an-existing-vpc-resource" >}}) from the cloud provider based just on its `id`.
```
Comment (justinvp) [inline content/blog/adopting-existing-cloud-resources-into-pulumi/index.md:27]: For `relref` links that have an anchor, there shouldn't be a `/` before the `#`.

```suggestion
* The `.get` methods available on every resource let you [get all the details for a resource]({{< relref "/docs/reference/pkg/aws/ec2/vpc#look-up-an-existing-vpc-resource" >}}) from the cloud provider based just on its `id`.
```

File: content/blog/announcing-pulumi-0.15-kubernetes-cicd-openstack-and-more/index.md (hunk 3 above)
Comment (jkisk) [reply content/blog/announcing-pulumi-0.15-kubernetes-cicd-openstack-and-more/index.md:197]: Agreed this is good change, but I think it may make sense to leave the two links pictured below as is, let me know if you think otherwise: 
image

File: content/blog/aws-cloudwatch-made-easy-with-pulumi-infrastructure-as-code/index.md (hunk 5 above)
Comment (jkisk) [reply content/blog/aws-cloudwatch-made-easy-with-pulumi-infrastructure-as-code/index.md:140]: I could also change the link text?  @justinvp  do you have an opinion here?

Comment (justinvp) [reply content/blog/aws-cloudwatch-made-easy-with-pulumi-infrastructure-as-code/index.md:140]: This link should not change. The original link is to the `awsx` README, which provides higher-level capabilities on top of `aws`, and this link is specifically about "Crosswalk for AWS" which is what `awsx` is.

Comment (jkisk) [reply content/blog/aws-cloudwatch-made-easy-with-pulumi-infrastructure-as-code/index.md]: Done, thank you both!

File: content/blog/announcing-pulumi-0.15-kubernetes-cicd-openstack-and-more/index.md (hunk 3 above)
Comment (jkisk) [reply content/blog/announcing-pulumi-0.15-kubernetes-cicd-openstack-and-more/index.md]: I ended up using the /docs/reference/pkg links consistently through page, as you suggested.

File: content/docs/guides/crosswalk/aws/vpc.md [hunk 7]
```diff
@@ -38,11 +38,11 @@ used for strong network isolation between endpoints and resources that are other
...
 ## Managing VPCs
 
-The [aws.ec2.Vpc]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#Vpc" >}}) class provides full access to the
-AWS VPC API, and [aws.ec2]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2" >}}) the entire AWS EC2 API. Using
+The [aws.ec2.Vpc]({{< relref "/docs/reference/pkg/aws/ec2/vpc/" >}}) class provides full access to the
+AWS VPC API, and [aws.ec2]({{< relref "/docs/reference/pkg/aws/ec2" >}}) the entire AWS EC2 API. Using
 these packages, you can configure all aspects of AWS networks for your applications and infrastructure.
 
-The [awsx.ec2.Vpc]({{< relref "/docs/reference/pkg/nodejs/pulumi/awsx/ec2#Vpc" >}}) class encapsulates a complete
+The [awsx.ec2.Vpc]({{< relref "/docs/reference/pkg/aws/ec2/vpc/" >}}) class encapsulates a complete
```
Comment (praneetloke) [inline content/docs/guides/crosswalk/aws/vpc.md:45]: This particular sentence is referring to `awsx`, so let's put this back to its previous value.

File: content/docs/guides/crosswalk/aws/vpc.md [hunk 8]
```diff
@@ -484,11 +484,11 @@ raw resource so that you can code directly to the underlying AWS resource types,
 
 For information about configuring each of these resources, please refer to each type's API documentation:
 
-* [Vpc]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#Vpc" >}})
-* [Subnet]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#Subnet" >}})
-* [InternetGateway]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#InternetGateway" >}})
-* [NatGateway]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#NatGateway" >}})
-* [SecurityGroup]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#SecurityGroup" >}})
+* [Vpc]({{< relref "/docs/reference/pkg/aws/ec2/vpc" >}})
```
Comment (praneetloke) [inline content/docs/guides/crosswalk/aws/vpc.md:487]: The line directly above this seems to talk about "API" documentation. I think these links are fine, but I wonder if we should update that line to say just "documentation"? @justinvp thoughts?

Comment (praneetloke) [reply content/docs/guides/crosswalk/aws/vpc.md:487]: There are other pages that use the same wording, so should we update them to read "resource documentation" or just leave them as "API documentation"? It's just a little ambiguous to me because I personally refer to the API docs as the language specific  API docs and the resource docs are..well, the resource API docs, perhaps?

File: content/docs/guides/crosswalk/aws/iam.md [hunk 9]
```diff
@@ -109,7 +109,7 @@ const rolePolicyAttachment = new aws.iam.RolePolicyAttachment("rpa", {
 ```
 
 For a full list of available managed policy ARNs, please refer to the
-[API documentation]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/iam" >}}).
+[API documentation]({{< relref "/docs/reference/pkg/aws/iam" >}}).
```
Comment (praneetloke) [inline content/docs/guides/crosswalk/aws/iam.md:112]: I am a little conflicted with the links we should have on this page. On the one hand having them link to the nodejs API docs (as they are today) makes sense because all of the code snippets are in TS. But I also like that we can link to a specific resource such as User, AccessKey UserPolicyAttachment under the IAM module, which lets the user see just the relevant resource. I am fine with keeping the changes you've made here but would like to see what @justinvp of this as well.

File: content/blog/run-your-own-rss-server/index.md [hunk 10]
```diff
@@ -34,7 +34,7 @@ As I mentioned, the service is packaged as a Docker container that runs on port
...
+We'll set these properties in the course of developing our program. The service itself, as a container, can easily be run on [AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/AWS_Fargate.html), so we can use the [`@pulumi/awsx`]({{< relref "/docs/reference/pkg/nodejs/pulumi/awsx" >}}) package to declare it, and for the database, we'll use [`@pulumi/aws`]({{< relref "//docs/reference/pkg/aws" >}}) to provision a small [RDS instance of PostgreSQL](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html). And finally, to make the Miniflux service publicly accessible (initially over HTTP; we'll switch to HTTPS later), we'll use an AWS [Network Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/network-load-balancers.html), since we'll only need to be able to route traffic by ports and protocols.
```
Comment (praneetloke) [inline content/blog/run-your-own-rss-server/index.md:37]: There seems to have been a find/replace error here. There's an extra leading slash.

```suggestion
We'll set these properties in the course of developing our program. The service itself, as a container, can easily be run on [AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/AWS_Fargate.html), so we can use the [`@pulumi/awsx`]({{< relref "/docs/reference/pkg/nodejs/pulumi/awsx" >}}) package to declare it, and for the database, we'll use [`@pulumi/aws`]({{< relref "/docs/reference/pkg/aws" >}}) to provision a small [RDS instance of PostgreSQL](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html). And finally, to make the Miniflux service publicly accessible (initially over HTTP; we'll switch to HTTPS later), we'll use an AWS [Network Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/network-load-balancers.html), since we'll only need to be able to route traffic by ports and protocols.
```

...but I think we should revert this change because we are referring to the `@pulumi/aws` npm package here, so maybe we should leave the link as it was before, to the nodejs API docs for `aws`?

File: content/blog/get-started-with-docker-on-aws-fargate-using-pulumi/index.md [hunk 11]
```diff
@@ -89,7 +89,7 @@ export const url = web.endpoint.hostname;
 ## Step 1. Create a Cluster
 
 The opening stanza imports Pulumi's
-[open source AWSX NPM package]({{< relref "/docs/reference/pkg/nodejs/pulumi/awsx" >}}), `@pulumi/awsx`. It
+[open source AWSX NPM package]({{< relref "/docs/guides/crosswalk/aws/" >}}), `@pulumi/awsx`. It
```
Comment (praneetloke) [inline content/blog/get-started-with-docker-on-aws-fargate-using-pulumi/index.md:92]: I think this should be reverted as it is talking about the `awsx` npm package specifically.

File: content/docs/guides/crosswalk/aws/vpc.md (hunk 7 above)
Comment (jkisk) [reply content/docs/guides/crosswalk/aws/vpc.md:45]: oh, thank you, good catch

File: content/docs/guides/crosswalk/aws/iam.md (hunk 9 above)
Comment (justinvp) [reply content/docs/guides/crosswalk/aws/iam.md:112]: Yeah, I'm a bit conflicted as well, but I do think the resource docs are a better representation of what we want to display to users first, and then they can click the link to the specific language API from there, so I'm fine with linking to those. Eventually these docs should be updated for more languages anyways (as the multi-lang effort progresses), so we'll eventually be linking to the resource docs first anyway.

File: content/docs/guides/crosswalk/aws/iam.md [hunk 12]
```diff
@@ -118,7 +118,7 @@ For a full list of available managed policy ARNs, please refer to the
 An AWS Identity and Access Management (IAM) user is an entity that you create in AWS to represent the person or
 application that uses it to interact with AWS. A user in AWS consists of a name and credentials.
 
-Use the [`aws.iam.User` class]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/iam#User" >}}) to create new
+Use the [`aws.iam.User` class]({{< relref "/docs/reference/pkg/aws/iam/user" >}}) to create new
```
Comment (justinvp) [inline content/docs/guides/crosswalk/aws/iam.md:121]: To @praneetloke's point, perhaps we should generalize the surrounding text a bit. Something like:

```suggestion
Use the [`User` resource]({{< relref "/docs/reference/pkg/aws/iam/user" >}}) to create new
```

File: content/docs/guides/crosswalk/aws/iam.md [hunk 13]
```diff
@@ -146,11 +146,11 @@ For more options available when configuring IAM users, please see the [API docum
 {{< relref "/docs/reference/pkg/nodejs/pulumi/aws/iam#User" >}}).
 
 If you'd like to configure non-service account users that can login to the
-AWS console, see [`UserLoginProfile`]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/iam#UserLoginProfile" >}}),
-and for creating access keys, see [`AccessKey`]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/iam#AccessKey" >}}).
+AWS console, see [`UserLoginProfile`]({{< relref "/docs/reference/pkg/aws/iam/userloginprofile" >}}),
+and for creating access keys, see [`AccessKey`]({{< relref "/docs/reference/pkg/aws/iam/accesskey" >}}).
 
 If you need to attach a managed policy ARN to your user, use the [`UserPolicyAttachment` class](
```
Comment (justinvp) [inline content/docs/guides/crosswalk/aws/iam.md:152]:
```suggestion
If you need to attach a managed policy ARN to your user, use the [`UserPolicyAttachment` resource](
```

File: content/docs/guides/crosswalk/aws/iam.md [hunk 14]
```diff
@@ -164,7 +164,7 @@ and needs administrator privileges, you can assign the appropriate permissions b
 Similarly, if a person changes jobs in your organization, instead of editing that user's permissions, you can remove
 them from the old groups and add them to the appropriate new groups.
 
-Use the [`aws.iam.Group` class]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/iam#Group" >}}) to manage
+Use the [`aws.iam.Group` class]({{< relref "/docs/reference/pkg/aws/iam/group" >}}) to manage
```
Comment (justinvp) [inline content/docs/guides/crosswalk/aws/iam.md:167]:
```suggestion
Use the [`Group` resource]({{< relref "/docs/reference/pkg/aws/iam/group" >}}) to manage
```

File: content/docs/guides/crosswalk/aws/iam.md [hunk 15]
```diff
@@ -199,10 +199,10 @@ const devTeam = new aws.iam.GroupMembership("dev-team", {
...
 
 For more information, please refer to the API documentation for [groups](
-{{< relref "/docs/reference/pkg/nodejs/pulumi/aws/iam#Group" >}}), [group membership](
-{{< relref "/docs/reference/pkg/nodejs/pulumi/aws/iam#GroupMembership" >}}), and [group policies](
-{{< relref "/docs/reference/pkg/nodejs/pulumi/aws/iam#GroupPolicy" >}}). If you need to attach a managed policy ARN to your group, use the
-[`GroupPolicyAttachment` class]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/iam#GroupPolicyAttachment" >}}).
+{{< relref "/docs/reference/pkg/aws/iam/group" >}}), [group membership](
+{{< relref "/docs/reference/pkg/aws/iam/groupmembership" >}}), and [group policies](
+{{< relref "/docs/reference/pkg/aws/iam/grouppolicy" >}}). If you need to attach a managed policy ARN to your group, use the
+[`GroupPolicyAttachment` class]({{< relref "/docs/reference/pkg/aws/iam/grouppolicyattachment" >}}).
```
Comment (justinvp) [inline content/docs/guides/crosswalk/aws/iam.md:205]:
```suggestion
[`GroupPolicyAttachment` resource]({{< relref "/docs/reference/pkg/aws/iam/grouppolicyattachment" >}}).
```

File: content/docs/guides/crosswalk/aws/iam.md [hunk 16]
```diff
@@ -215,7 +215,7 @@ in AWS. Instead of being uniquely associated with one person, however, a role is
 a role does not have standard long-term credentials such as a password or access keys associated with it. Instead, when
 you assume a role, it provides you with temporary security credentials for your role session.
 
-To manage IAM roles, use the [`aws.iam.Role` class]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/iam#Role" >}}).
+To manage IAM roles, use the [`aws.iam.Role` class]({{< relref "/docs/reference/pkg/aws/iam/role" >}}).
```
Comment (justinvp) [inline content/docs/guides/crosswalk/aws/iam.md:218]:
```suggestion
To manage IAM roles, use the [`Role` resource]({{< relref "/docs/reference/pkg/aws/iam/role" >}}).
```

File: content/docs/guides/crosswalk/aws/iam.md [hunk 17]
```diff
@@ -244,19 +244,19 @@ Roles are often useful for creating [instance profiles](
 https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html), which
 controls the IAM role assumed by compute running inside of your AWS account, whether that be in EC2, ECS, EKS, or
 Lambda, for example. To create one, use the [`InstanceProfile` class](
```
Comment (justinvp) [inline content/docs/guides/crosswalk/aws/iam.md:246]:
```suggestion
Lambda, for example. To create one, use the [`InstanceProfile` resource](
```

File: content/docs/guides/crosswalk/aws/other.md [hunk 18]
```diff
@@ -52,105 +52,105 @@ tbody tr td:first-child {
...
-| [Amazon Simple Notification Service (SNS)](https://aws.amazon.com/sns/) | Managed pub/sub messaging | | [sns]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/sns" >}}) |
-| [Amazon Simple Queue Service (SQS)](https://aws.amazon.com/sqs/) | Managed, reliable queuing | | [sqs]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/sqs" >}}) |
-| [AWS Systems Manager](https://docs.aws.amazon.com/systems-manager/) | Automate management tasks | | [ssm]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ssm" >}}) |
-| [AWS Storage Gateway](https://aws.amazon.com/storagegateway/) | Hybrid cloud storage | | [storagegateway]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/storagegateway" >}}) |
-| [Amazon Simple Workflow Service (SWF)](https://aws.amazon.com/swf/) | Background jobs with steps | | [swf]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/swf" >}}) |
-| [AWS Transfer for SFTP](https://aws.amazon.com/sftp/) | Managed SFTP service | | [transfer]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/transfer" >}}) |
+| [AWS Certificate Manager](https://aws.amazon.com/certificate-manager/) | Manage TLS/SSL certificates | | [acm]({{< relref "/docs/reference/pkg/aws/acm " >}}) |
```
Comment (justinvp) [inline content/docs/guides/crosswalk/aws/other.md:55]:
```suggestion
| [AWS Certificate Manager](https://aws.amazon.com/certificate-manager/) | Manage TLS/SSL certificates | | [acm]({{< relref "/docs/reference/pkg/aws/acm" >}}) |
```

File: content/docs/guides/crosswalk/aws/vpc.md [hunk 19]
```diff
@@ -38,8 +38,8 @@ used for strong network isolation between endpoints and resources that are other
 
 ## Managing VPCs
 
-The [aws.ec2.Vpc]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#Vpc" >}}) class provides full access to the
-AWS VPC API, and [aws.ec2]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2" >}}) the entire AWS EC2 API. Using
+The [aws.ec2.Vpc]({{< relref "/docs/reference/pkg/aws/ec2/vpc/" >}}) class provides full access to the
```
Comment (justinvp) [inline content/docs/guides/crosswalk/aws/vpc.md:41]:
```suggestion
The [Vpc resource]({{< relref "/docs/reference/pkg/aws/ec2/vpc" >}}) provides full access to the
```

File: content/docs/guides/crosswalk/aws/vpc.md (hunk 8 above)
Comment (justinvp) [reply content/docs/guides/crosswalk/aws/vpc.md:487]: I think it's fine to say `API documentation` as we're keeping all these under the `API Reference` section in the table of contents.

File: content/docs/guides/crosswalk/aws/vpc.md [hunk 20]
```diff
@@ -484,11 +484,11 @@ raw resource so that you can code directly to the underlying AWS resource types,
...
 For information about configuring each of these resources, please refer to each type's API documentation:
 
-* [Vpc]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#Vpc" >}})
-* [Subnet]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#Subnet" >}})
-* [InternetGateway]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#InternetGateway" >}})
-* [NatGateway]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#NatGateway" >}})
-* [SecurityGroup]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#SecurityGroup" >}})
+* [Vpc]({{< relref "/docs/reference/pkg/aws/ec2/vpc" >}})
+* [Subnet]({{< relref "/docs/reference/pkg/aws/ec2/subnet" >}})
+* [InternetGateway]({{< relref "/docs/reference/pkg/aws/ec2/internetgateway/" >}})
```
Comment (justinvp) [inline content/docs/guides/crosswalk/aws/vpc.md:489]:
```suggestion
* [InternetGateway]({{< relref "/docs/reference/pkg/aws/ec2/internetgateway" >}})
```

File: content/docs/guides/crosswalk/aws/vpc.md [hunk 21]
```diff
@@ -484,11 +484,11 @@ raw resource so that you can code directly to the underlying AWS resource types,
...
+* [NatGateway]({{< relref "/docs/reference/pkg/aws/ec2/natgateway/" >}})
```
Comment (justinvp) [inline content/docs/guides/crosswalk/aws/vpc.md:490]:
```suggestion
* [NatGateway]({{< relref "/docs/reference/pkg/aws/ec2/natgateway" >}})
```

File: content/docs/guides/crosswalk/aws/vpc.md [hunk 22]
```diff
@@ -484,11 +484,11 @@ raw resource so that you can code directly to the underlying AWS resource types,
...
+* [SecurityGroup]({{< relref "/docs/reference/pkg/aws/ec2/securitygroup/" >}})
```
Comment (justinvp) [inline content/docs/guides/crosswalk/aws/vpc.md:491]:
```suggestion
* [SecurityGroup]({{< relref "/docs/reference/pkg/aws/ec2/securitygroup" >}})
```

**Comment by cnunciato** [comment]
> In addition, for the relref's I think should also end with a trailing /, IFF there isn't an anchor link.

This is not required actually.
